api.bs

```diff
@@ -1157,17 +1157,22 @@ nullable integer |attributedValueForSingleEpochOpt|:
 1.  Let |currentValue| be the result of [=map/get|getting the value=] of |key|
     in the [=privacy budget store=].
 
-1.  Let |singleEpochQuery| be true if |attributedValueForSingleEpochOpt| is non-null, false otherwise.
+1.  Let |sensitivity| be |l1Norm| if |l1Norm| is non-null, |value| otherwise.
```

Comment:
We shouldn't call this value `sensitivity`, unless we make it `2 * |value|`. With that change this PR LGTM, which also simplifies things (avoids another branch on line 1166)

Note that I think this is equivalent to what I had before @alexanderknop 's review, so I'll let him argue for his POV :)

Reply:
```suggestion
1.  Let |sensitivity| be |l1Norm| if |l1Norm| is non-null, |value| * 2 otherwise.
```